src/modules/commander/commander_params.c

```diff
@@ -818,11 +823,12 @@ PARAM_DEFINE_INT32(COM_TAKEOFF_ACT, 0);
 PARAM_DEFINE_INT32(NAV_DLL_ACT, 0);
 
 /**
- * Set RC loss failsafe mode
+ * Set RC loss failsafe action (mode).
  *
- * The RC loss failsafe will only be entered after a timeout,
- * set by COM_RC_LOSS_T in seconds. If RC input checks have been disabled
- * by setting the COM_RC_IN_MODE param it will not be triggered.
+ * The RC loss failsafe will only be entered if the RC connection is not recovered before first the
+ * RC signal loss theshold timeout (COM_RC_LOSS_T) and then the RC connection lost timeout (NAV_RCL_ACT) have triggered.
+ * The action will not be triggered if RC input checks have been disabled (COM_RC_IN_MODE)
+ * or if RC loss exceptions are disabled for the current mode (COM_RCL_EXCEPT).
```

Comment:
Updated to match the rest, and to note the existence of COM_RCL_EXCEPT

Reply:
```suggestion
 * or if there's an exception for the current mode in COM_RCL_EXCEPT.
```